### PR TITLE
Re-structure, re-design and update the README.md

### PR DESCRIPTION
**Pull request by abhisharma404**
## Status
**READY**

## Description
Re-structure, re-design and update the README.md.

This PR does the following:

- [x] GUI preview deployed to Heroku
- [x] Added Heroku badge
- [x] Re-structured and re-design the `README.md` for elegance styling
- [x] Removed un-necessary information
- [x] Added two installation methods
- [x] Corrected hyper-links
- [x] Made absolute links to relative links
- [x] Updated setup GIF
- [x] New dashboard image

**Preview:** https://github.com/abhisharma404/SecureTea-Project/tree/final-readme

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Todos
- [ ] Tests
- [x] Documentation

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

```sh
git pull --prune
git checkout <feature-branch>
```

## Impacted Areas in Application
List general components of the application that this PR will affect:
1. `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,63 +31,65 @@
...
-![Setup Intro][Setup-help-animation]
+- [x] [Intrusion Detection System](/doc/en-US/user_guide.md#intrusion-detection-system)
+- [x] [Firewall](/doc/en-US/user_guide.md#firewall)
+- [x] [AntiVirus](/doc/en-US/user_guide.md#antivirus)
+- [x] [Server Log Monitor](/doc/en-US/user_guide.md#server-log-monitor)
+- [x] [System Log Monitor](/doc/en-US/user_guide.md#system-log-monitor)
+- [x] [Local Web Deface Detection & Prevention System](/doc/en-US/user_guide.md#web-deface-detection)
+- [x] [Auto Web Server Patcher](/doc/en-US/user_guide.md#auto-server-patcher)
+- [x] [Insecure Headers Detection](/doc/en-US/user_guide.md#insecure-headers)
+- [x] [IoT Anonymity checker](/doc/en-US/user_guide.md#iot-anonymity-checker)
```
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: @rejahrehim Hi there, Can you explain with an example, that would be a lot helpful :) I'm not able to understand what casing here refers to, lower case & upper case?

[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: @rejahrehim Do you mean to rename "Intrusion **D**etection **S**ystem" to "Intrusion **d**etection **s**ystem" and son on...?

[the remark this answers is not in the document]
Comment (rejahrehim) [reply README.md]: IoT Anonymity checker -> IoT Anonymity Checker
All entries are in title case format only some are not

File: README.md [hunk 2]
```diff
@@ -14,63 +31,65 @@
...
+- [x] [Auto report generation using OSINT](/doc/en-US/user_guide.md)
```
[the remark this answers is not in the document]
Comment (rejahrehim) [reply README.md]: Auto report generation using OSINT  -> Auto Report Generation Using OSINT

https://titlecaseconverter.com/ Use Chicago style

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: Done :)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: Done :)

File: README.md [hunk 3]
```diff
@@ -14,63 +31,65 @@
...
+- [x] [Notifying suspicious activities using various mediums (Twitter, Telegram, Slack, Gmail, SMS, AWS & more)](doc/en-US/user_guide.md#setting-up-notifiers)
+- [x] [Interactive GUI for ease of setting up](/doc/en-US/user_guide.md#configuring-using-web-ui)
```
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: Done :)

File: README.md [hunk 4]
```diff
@@ -14,63 +31,65 @@
...
+- [x] [Firewall](/doc/en-US/user_guide.md#firewall)
+- [x] [AntiVirus](/doc/en-US/user_guide.md#antivirus)
+- [x] [Server Log Monitor](/doc/en-US/user_guide.md#server-log-monitor)
+- [x] [System Log Monitor](/doc/en-US/user_guide.md#system-log-monitor)
+- [x] [Local Web Deface Detection & Prevention System](/doc/en-US/user_guide.md#web-deface-detection)
+- [x] [Auto Web Server Patcher](/doc/en-US/user_guide.md#auto-server-patcher)
+- [x] [Insecure Headers Detection](/doc/en-US/user_guide.md#insecure-headers)
+- [x] [IoT Anonymity checker](/doc/en-US/user_guide.md#iot-anonymity-checker)
+- [x] [Auto report generation using OSINT](/doc/en-US/user_guide.md)
+- [x] [Notifying suspicious activities using various mediums (Twitter, Telegram, Slack, Gmail, SMS, AWS & more)](doc/en-US/user_guide.md#setting-up-notifiers)
```
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: Done :)

File: README.md [hunk 5]
```diff
@@ -14,63 +31,65 @@
...
+
+## Installation
 
-## Table Of Contents
--   [Pre-requisites](/doc/en-US/user_guide.md#pre-requisites)
--   [Installation Procedure](#installation-procedure)
--   [Quick Start](#quick-start)
--   [Suggestions and Contribution](#suggestions-and-contributing)
--   [Code Of Conduct](/CODE_OF_CONDUCT.md)
--   [User guide](/doc/en-US/user_guide.md)
```
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: This part was removed :)

File: README.md [hunk 6]
```diff
@@ -14,63 +31,65 @@
...
--   [Developer guide](/doc/en-US/dev_guide.md)
```
[the remark this answers is not in the document]
Comment (abhisharma404) [reply README.md]: This part was also removed :)

**Pull request merged**